components/plurals/tests/plurals.rs

```diff
@@ -15,6 +17,23 @@ fn test_plural_rules() {
     assert_eq!(pr.select(5_usize), PluralCategory::Other);
 }
 
+#[test]
+fn test_plural_rules_custom() {
```

Comment:
what is this test for and why is it not data-driven? (it's ok if you just add a comment explaining its purpose for 0.1).

Reply:
This was a leftover from when I was trying to figure out whether I'm loading things correctly. Removed.